src/components/forms/Scopes/modules/PermissionGroup/PermissionGroup.view.vue

```diff
@@ -6,7 +6,7 @@
       </p>
 
       <v-hover
-        v-slot:default="{ isHover }"
+        #default="{ isHover }"
```

Comment:
What means `#`?

Reply:
https://ru.vuejs.org/v2/guide/components-slots.html#%D0%A1%D0%BE%D0%BA%D1%80%D0%B0%D1%89%D1%91%D0%BD%D0%BD%D0%B0%D1%8F-%D0%B7%D0%B0%D0%BF%D0%B8%D1%81%D1%8C-%D0%B4%D0%BB%D1%8F-%D0%B8%D0%BC%D0%B5%D0%BD%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D1%85-%D1%81%D0%BB%D0%BE%D1%82%D0%BE%D0%B2